cytodynamics-test-container/src/test/java/com/linkedin/cytodynamics/test/TestDynamicLoad.java

```diff
@@ -139,12 +141,43 @@ public void testLoadB() throws Exception {
     assertEquals(implementation.getValue(), "B");
   }
 
+  @Test(description = "Given that the loader is a parent of another classloader, then it should still properly do "
```

Comment:
Do you think it's worth testing the `loadClass(name, resolve)` impl instead of just the `loadClass(name)` since the former is what is being overridden? Specifically the case where `resolve == true` since `loadClass(name)` already tests `loadClass(name, resolve=false)`.

Reply:
Yep, `loadClass(name, resolve)` can be considered part of the API, so it can be tested. Thanks for pointing that out.